nut/rootfs/etc/cont-init.d/nut.sh

```
@@ -94,12 +106,17 @@ if bashio::config.equals 'mode' 'netserver' ;then
             echo "  port = ${upsport}"
         } >> /etc/nut/ups.conf
 
+        debug_min_set='n'
         OIFS=$IFS
         IFS=$'\n'
         for configitem in $(bashio::config "devices[${device}].config"); do
             echo "  ${configitem}" >> /etc/nut/ups.conf
+            [[ "${configitem}" =~ ^debug_min\ *= ]] && debug_min_set='y'
         done
         IFS="$OIFS"
+        if [ "${debug_min_set}" = 'n' -a $debug_min -ge 1 ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)